core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProvider.scala

```diff
@@ -150,7 +150,10 @@ class CompactEntityProvider(requestArguments: EntityRequestArguments, repository
     }
   }
 
-  override def listEntities(entityType: String): Source[Entity, NotUsed] = ???
+  override def listEntities(entityType: String): Source[Entity, NotUsed] =
+    throw new DataEntityException("list all entities not supported for compact data tables.",
+                                  code = StatusCodes.NotImplemented
+    )
```

Comment:
`DataEntityException` status codes are bubbled up through the API responses, so this helps make the unit test accurate. When it was just `???`, the API response ended up as a 500. 500s are too common so I couldn't be certain why it was being thrown.